packages/react-cookie/src/CookieUniversalProvider.tsx

```diff
@@ -1,6 +1,6 @@
 import React from 'react';
-import {useNetworkManager} from '@shopify/react-network';
 import {useLazyRef} from '@shopify/react-hooks';
+import {useNetworkManager} from '@shopify/react-network';
```

Comment:
Why does this kind of re-order need to happen? Aren't they both external?

Reply:
Yeah good question, this looks like a mistake.